environment.yml

```diff
@@ -13,11 +13,10 @@ dependencies:
     - packaging
```

Comment:
Minor recommendation, but could we change `defaults` to `nodefaults` at L4? I think all of the packages should be on `conda-forge` channel now, and changing to `nodefaults` will make the `conda`/`mamba` solvers faster since it doesn't have to look through an extra channel.

Reply:
Sounds good. DOne in 1badc83e241eda937521ecbd5dd669dd2cab5474.